www/front_src/src/components/navigation/navigation.scss

```diff
@@ -823,4 +823,11 @@ a {
   .sidebarCollapse span {
     display: none;
   }
+}
+@media screen and(max-width: 1400px) {
+  li.menu-item:last-child .collapsed-items li.collapsed-item:first-child > ul.collapsed-level-items {
```

Comment:
can you check the indentation please ;)

Reply:
Can you be specific please :)? This is the normal CSS indentations. ;)

Reply:
indeed... I thought that the indentation was wrong, but it's not... sorry